"gradlew distribution" failed on "branch-0.5" release source code

**Cannot invoke method log() on null object** error while building Pravega from release `branch-0.5` source code (tar & zip) files 
https://github.com/pravega/pravega/archive/branch-0.5.tar.gz
https://github.com/pravega/pravega/archive/branch-0.5.zip

**gradlew distribution log**
```
[root@master pravega-branch-0.5]# ./gradlew distribution

> Configure project :
Build JVM Version is : 1.8.0_181 (Oracle Corporation 25.181-b13)
No git repository found for :. Accessing grgit will cause an NPE.

FAILURE: Build failed with an exception.

* Where:
Build file '/root/workdir/pravega-branch-0.5/build.gradle' line: 918

* What went wrong:
A problem occurred evaluating root project 'pravega'.
> Cannot invoke method log() on null object

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
```

**gradlew distribution --info log**
```
[root@master pravega-branch-0.5]# ./gradlew distribution --info
Initialized native services in: /root/.gradle/native
The client will now receive all logging from the daemon (pid: 28402). The daemon log file: /root/.gradle/daemon/4.10.2/daemon-28402.out.log
Starting 16th build in daemon [uptime: 1 hrs 19 mins 12.216 secs, performance: 99%, no major garbage collections]
Using 4 worker leases.
Starting Build
Settings evaluated using settings file '/root/workdir/pravega-branch-0.5/settings.gradle'.
Projects loaded. Root project using build file '/root/workdir/pravega-branch-0.5/build.gradle'.
Included projects: [root project 'pravega', project ':bindings', project ':client', project ':common', project ':controller', project ':segmentstore', project ':shared', project ':standalone', project ':test', project ':segmentstore:contracts', project ':segmentstore:server', project ':segmentstore:storage', project ':shared:authplugin', project ':shared:cluster', project ':shared:controller-api', project ':shared:metrics', project ':shared:protocol', project ':test:integration', project ':test:system', project ':test:testcommon', project ':segmentstore:server:host', project ':segmentstore:storage:impl']

> Configure project :
Evaluating root project 'pravega' using build file '/root/workdir/pravega-branch-0.5/build.gradle'.
Build JVM Version is : 1.8.0_181 (Oracle Corporation 25.181-b13)
No git repository found for :. Accessing grgit will cause an NPE.

FAILURE: Build failed with an exception.

* Where:
Build file '/root/workdir/pravega-branch-0.5/build.gradle' line: 918

* What went wrong:
A problem occurred evaluating root project 'pravega'.
> Cannot invoke method log() on null object

* Try:
Run with --stacktrace option to get the stack trace. Run with --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 0s
```

@fpj  @shrids I have observed same issue too. RC builds of 0.4.0 are working fine while doing `./gradlew install` where `branch-0.5` contents building (same method) fails.
```
[root@manager1 pravega-branch-0.5]# ./gradlew install

> Configure project :
Build JVM Version is : 1.8.0_181 (Oracle Corporation 25.181-b13)
No git repository found for :. Accessing grgit will cause an NPE.

FAILURE: Build failed with an exception.

* Where:
Build file '/home/sumit/pravega-050-fc-bld-2222/pravega-0.5.0-2222.8ba5431/pravega-branch-0.5/build.gradle' line: 918

* What went wrong:
A problem occurred evaluating root project 'pravega'.
> Cannot invoke method log() on null object

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
```

--- a/build.gradle
+++ b/build.gradle
@@ -913,7 +913,7 @@ project('test:system') {
 
 def getProjectVersion() {
     String ver = pravegaVersion
-    if (ver.contains("-SNAPSHOT")) {
+    if (grgit && ver.contains("-SNAPSHOT")) {
         String versionLabel = ver.substring(0, ver.indexOf("-SNAPSHOT"))
         def count = grgit.log(includes:['HEAD']).size()
         def commitId = "${grgit.head().abbreviatedId}"

